pkg/registry/resource/resourceclaim/strategy.go

```diff
@@ -76,9 +90,11 @@ func (resourceclaimStrategy) PrepareForCreate(ctx context.Context, obj runtime.O
 	dropDisabledFields(claim, nil)
 }
 
-func (resourceclaimStrategy) Validate(ctx context.Context, obj runtime.Object) field.ErrorList {
+func (s resourceclaimStrategy) Validate(ctx context.Context, obj runtime.Object) field.ErrorList {
```

Comment:
Update all receivers to use pointers.

```suggestion
func (s *resourceclaimStrategy) Validate(ctx context.Context, obj runtime.Object) field.ErrorList {
```